File: ServerBackend/ServerBackend/Player.cs
using System.Device.Location;
using Newtonsoft.Json;
using Red;

namespace ServerBackend
{
    class Player
    {
        public Player(string id, WebSocketDialog wsd)
        {
            Id = id;
            Wsd = wsd;
        }

        public string Id { get; set; }
        public string DisplayName { get; set; } = "[Unnamed]";
        public int Team { get; set; } = 0;
        
        [JsonIgnore]
        public GeoCoordinate Location { get; set; } = GeoCoordinate.Unknown;

        public bool Ready { get; set; } = false;

        [JsonIgnore] 
        public WebSocketDialog Wsd;
    }
}

File: MultiplayerAugmentedRealityKit/app/src/main/java/dk/aau/sw805f18/ar/models/Player.java
package dk.aau.sw805f18.ar.models;

public class Player {
    public String Id;
    public String DisplayName;
    public int Team;
    public boolean Ready;

    public Player(String id, String displayName, int team, boolean ready) {
        Id = id;
        DisplayName = displayName;
        Team = team;
        Ready = ready;
    }
    public Player() {

    }
}
